Seldon Docs failing on readthedocs

<!-- Welcome and thank you for helping us make Seldon Core better!

To help us address your issue, please provide us as much of the information requested below as possble. Thanks! -->

```
Running Sphinx v2.4.4
fatal: no tag exactly matches '4eda2e53b77a63c3c3ae8e9f96dbac491e0949ef'
loading translations [en]... done
/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/m2r.py:652: RemovedInSphinx30Warning: app.add_source_parser() does not support suffix argument. Use app.add_source_suffix() instead.
  app.add_source_parser('.md', M2RParser)Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/sphinx/cmd/build.py", line 275, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/sphinx/application.py", line 245, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/sphinx/application.py", line 397, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/sphinx/registry.py", line 453, in load_extension
    mod = import_module(extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbsphinx.py", line 41, in <module>
    import nbconvert
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbconvert/__init__.py", line 4, in <module>
    from .exporters import *
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbconvert/exporters/__init__.py", line 4, in <module>
    from .slides import SlidesExporter
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbconvert/exporters/slides.py", line 12, in <module>
    from ..preprocessors.base import Preprocessor
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbconvert/preprocessors/__init__.py", line 7, in <module>
    from .csshtmlheader import CSSHTMLHeaderPreprocessor
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/nbconvert/preprocessors/csshtmlheader.py", line 14, in <module>
    from jupyterlab_pygments import JupyterStyle
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/jupyterlab_pygments/__init__.py", line 4, in <module>
    from .style import JupyterStyle
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/jupyterlab_pygments/style.py", line 10, in <module>
    class JupyterStyle(Style):
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/pygments/style.py", line 101, in __new__
    ndef[0] = colorformat(styledef)
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/pygments/style.py", line 58, in colorformat
    assert False, "wrong color format %r" % text
AssertionError: wrong color format 'var(--jp-mirror-editor-variable-color)'Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/seldon-core/envs/latest/lib/python3.6/site-packages/pygments/style.py", line 58, in colorformat
    assert False, "wrong color format %r" % text
AssertionError: wrong color format 'var(--jp-mirror-editor-variable-color)'
The full traceback has been saved in /tmp/sphinx-err-9zwqoue6.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```
Discussed at https://github.com/voila-dashboards/voila/issues/224#issuecomment-513349992
with possible commit that solves

--- a/doc/requirements_docs.txt
+++ b/doc/requirements_docs.txt
@@ -8,3 +8,4 @@ nbsphinx>=0.4.2
 nbsphinx-link>=1.2.0
 ipykernel>=5.1.0
 ipython>=7.2.0
+pygments>=2.4.1,<3

